<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatWindows</class>
 <widget class="QWidget" name="chatWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: #fff; /* 背景色 */
color: #333; /* 文字颜色 */
border: 1px solid #ccc; /* 边框 */
border-radius: 5px; /* 圆角 */
padding: 5px; /* 内边距 */
}

QPushButton {
background-color: #4CAF50; /* 按钮背景色 */
color: white; /* 文字颜色 */
border: none; /* 去掉按钮边框 */
padding: 8px 16px; /* 按钮内边距 */
border-radius: 3px; /* 圆角 */
}

QPushButton:hover {
background-color: #45a049; /* 鼠标悬停时的背景色 */
}

QTextEdit {
background-color: #f0f0f0; /* 输入框背景色 */
border: 1px solid #ccc; /* 输入框边框 */
padding: 5px; /* 输入框内边距 */
border-radius: 3px; /* 圆角 */
}

QLabel {
color: #555; /* 标签文字颜色 */
padding: 5px; /* 标签内边距 */
}


</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>输入框</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>711</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>发送消息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>聊天框</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pengyouquanbiaoqing.png</normaloff>:/pengyouquanbiaoqing.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/wenjianchuanshu.png</normaloff>:/wenjianchuanshu.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/yanseqiehuan.png</normaloff>:/yanseqiehuan.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>570</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ai智能回复</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/geshi.png</normaloff>:/geshi.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>570</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择Ai回复模型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>600</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>711</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
